src/main/java/org/fundacionjala/coding/yury/SpinWord.java

```diff
@@ -0,0 +1,66 @@
+package org.fundacionjala.coding.yury;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Created by YuryOrtuno on 5/23/2017.
+ */
+public class SpinWord {
+    public static final int REVERSE_LENGTH = 5;
+    private List<String> wordList;
+
+    /**
+     * The default constructor.
+     */
+    public SpinWord() {
+        wordList = new ArrayList<>();
```

Comment:
For this solution you don't need a List as attribute of class.
Remove that dependency for your solution